### DISCUSSION TITLE
PR's not appearing

### DISCUSSION THREAD

**Discussion by LCativiela** [community/community#128150] [Pull Requests]
### Select Topic Area

Bug

### Body

Hey! 
I have three workers doing pull requests on a project, however, one of them pushes the PR but it never appears on my page, accesing through link results in a 404.
I have seen him do it via screen share, and the ID of the PR gets increased, so the next one I do jumps a number. He can see them in his screen

Does this rign any bells? I have checked the project configuration and he is a collaborator, and has no rule blocking him from doing this PR's

Thanks for your help!

**Comment by Vishwa-ud** [comment]
It sounds like there might be a couple of potential issues causing this problem. Here are a few things to consider:
Check Branch Permissions : Ensure that the branch the worker is pushing the pull request to is not protected or restricted in a way that prevents them from creating pull requests. Sometimes, repository settings can limit who can create pull requests against certain branches.
Verify Repository Permissions: Double-check the repository permissions to confirm that the worker has the necessary permissions to create pull requests. Even if they are a collaborator, there could be specific permissions settings that are affecting their ability to create pull requests.
